src/csharp/Microsoft.Spark.Worker/Processor/BroadcastVariableProcessor.cs

```diff
@@ -54,18 +57,34 @@ internal BroadcastVariables Process(Stream stream)
                     }
                     else
                     {
-                        string path = SerDe.ReadString(stream);
-                        // TODO: Register new broadcast variable.
+                        var path = SerDe.ReadString(stream);
+                        FileStream fStream = File.Open(path, FileMode.Open, FileAccess.Read);
+                        formatter.Binder = new DeserializationBinder();
+                        var value = formatter.Deserialize(fStream);
```

Comment:
Please check the coding guideline for `var` usage.

Reply:
Since the type is not known at compile time here and is determined during runtime, I think `var` needs to be used here.

Reply:
Should we use `object` here?